### PR TITLE
Add Base Sepolia config

### PR DESCRIPTION
**Pull request by 0xmichalis**
### Backward compatibility

Yes

### Testing

Currently failing to relay a message from Alfajores to Base Sepolia:
<details>

```console
$ hyperlane send message --relay
Hyperlane CLI
? Select the origin chain alfajores
? Select the destination chain basesepolia
Running pre-flight checks for chains...
✅ Chains are valid
✅ Signer is valid
✅ Balances are sufficient
Dispatching message
Pending https://alfajores.celoscan.io/tx/0x83350a77b24778d865cb77182848db939a85e78eb81062d5bd42c97a877a8cba (waiting 1 blocks for confirmation)
Sent message from alfajores to 0xb7A35D65b478Ab08F3570838B6BbaD324692583e on basesepolia.
Message ID: 0xe1c11f5128fefd7fc96ed4b61311ff89cae4c8a5149a15beb68b78b4dc22509c
Message:
    parsed:
      version: 3
      nonce: 2032
      origin: 44787
      sender: "0x000000000000000000000000a90ede59ec974adfa6f6a1f7acfd1d42089d8f23"
      destination: 84532
      recipient: "0x000000000000000000000000b7a35d65b478ab08f3570838b6bbad324692583e"
      body: "0x48656c6c6f21"
      originChain: alfajores
      destinationChain: basesepolia
    id: "0xe1c11f5128fefd7fc96ed4b61311ff89cae4c8a5149a15beb68b78b4dc22509c"
    message: "0x03000007f00000aef3000000000000000000000000a90ede59ec974adfa6f6a1f7a\
      cfd1d42089d8f2300014a34000000000000000000000000b7a35d65b478ab08f3570838b6bbad\
      324692583e48656c6c6f21"
    
Attempting self-relay of message
All providers failed for method call and params {
  "transaction": {
    "to": "0xeee0A8f87D358672708598714C84f3E2B741019c",
    "data": "0xe445e7dd",
    "accessList": null
  },
  "blockTag": "latest"
}
All providers failed for method call and params {
  "transaction": {
    "to": "0xeee0A8f87D358672708598714C84f3E2B741019c",
    "data": "0xe445e7dd",
    "accessList": null
  },
  "blockTag": "latest"
}
All providers failed for method call and params {
  "transaction": {
    "to": "0xeee0A8f87D358672708598714C84f3E2B741019c",
    "data": "0xe445e7dd",
    "accessList": null
  },
  "blockTag": "latest"
}
All providers failed for method call and params {
  "transaction": {
    "to": "0xeee0A8f87D358672708598714C84f3E2B741019c",
    "data": "0xe445e7dd",
    "accessList": null
  },
  "blockTag": "latest"
}
All providers failed for method call and params {
  "transaction": {
    "to": "0xeee0A8f87D358672708598714C84f3E2B741019c",
    "data": "0xe445e7dd",
    "accessList": null
  },
  "blockTag": "latest"
}
All providers failed for method call and params {
  "transaction": {
    "to": "0xeee0A8f87D358672708598714C84f3E2B741019c",
    "data": "0xe445e7dd",
    "accessList": null
  },
  "blockTag": "latest"
}
All providers failed for method call and params {
  "transaction": {
    "to": "0xeee0A8f87D358672708598714C84f3E2B741019c",
    "data": "0xe445e7dd",
    "accessList": null
  },
  "blockTag": "latest"
}
All providers failed for method call and params {
  "transaction": {
    "to": "0xeee0A8f87D358672708598714C84f3E2B741019c",
    "data": "0xe445e7dd",
    "accessList": null
  },
  "blockTag": "latest"
}
All providers failed for method call and params {
  "transaction": {
    "to": "0xeee0A8f87D358672708598714C84f3E2B741019c",
    "data": "0xe445e7dd",
    "accessList": null
  },
  "blockTag": "latest"
}
All providers failed for method call and params {
  "transaction": {
    "to": "0xeee0A8f87D358672708598714C84f3E2B741019c",
    "data": "0xe445e7dd",
    "accessList": null
  },
  "blockTag": "latest"
}
All providers failed for method call and params {
  "transaction": {
    "to": "0xeee0A8f87D358672708598714C84f3E2B741019c",
    "data": "0xe445e7dd",
    "accessList": null
  },
  "blockTag": "latest"
}
All providers failed for method call and params {
  "transaction": {
    "to": "0xeee0A8f87D358672708598714C84f3E2B741019c",
    "data": "0xe445e7dd",
    "accessList": null
  },
  "blockTag": "latest"
}
All providers failed for method call and params {
  "transaction": {
    "to": "0x221FA9CBaFcd6c1C3d206571Cf4427703e023FFa",
    "data": "0xe445e7dd",
    "accessList": null
  },
  "blockTag": "latest"
}
All providers failed for method call and params {
  "transaction": {
    "to": "0x221FA9CBaFcd6c1C3d206571Cf4427703e023FFa",
    "data": "0xe445e7dd",
    "accessList": null
  },
  "blockTag": "latest"
}
All providers failed for method call and params {
  "transaction": {
    "to": "0x221FA9CBaFcd6c1C3d206571Cf4427703e023FFa",
    "data": "0xe445e7dd",
    "accessList": null
  },
  "blockTag": "latest"
}
Encountered error sending message from alfajores to basesepolia

hyperlane send message

Send a test message to a remote chain

Options:
      --log                                Log output format
                                            [string] [choices: "pretty", "json"]
      --verbosity                          Log verbosity level
            [string] [choices: "trace", "debug", "info", "warn", "error", "off"]
  -r, --registry                           Registry URI, such as a Github repo U
                                           RL or a local file path
       [string] [default: "https://github.com/hyperlane-xyz/hyperlane-registry"]
      --overrides                          Path to a local registry to override
                                           the default registry
                                 [string] [default: "/home/michalis/.hyperlane"]
  -k, --key, --private-key, --seed-phrase  A hex private key or seed phrase for
                                           transaction signing, or use the HYP_K
                                           EY env var.
                                         [string] [default: process.env.HYP_KEY]
  -y, --yes                                Skip confirmation prompts
                                                      [boolean] [default: false]
      --help                               Show help                   [boolean]
      --origin                             Origin chain to send message from
                                                                        [string]
      --destination                        Destination chain to send message to
                                                                        [string]
      --timeout                            Timeout in seconds
                                                         [number] [default: 300]
      --quick                              Skip wait for message to be delivered
                                                      [boolean] [default: false]
      --relay                              Handle self-relay of message on desti
                                           nation chain
                                                      [boolean] [default: false]
      --body                               Optional Message body
                                                    [string] [default: "Hello!"]

Error: missing revert data in call exception; Transaction reverted without a reason string [ See: https://links.ethers.org/v5-errors-CALL_EXCEPTION ] (data="0x", transaction={"to":"0x221FA9CBaFcd6c1C3d206571Cf4427703e023FFa","data":"0xe445e7dd","accessList":null}, error={"reason":"processing response error","code":"SERVER_ERROR","body":"{\"jsonrpc\":\"2.0\",\"id\":121,\"error\":{\"code\":-32000,\"message\":\"execution reverted\"}}\n","error":{"code":-32000},"requestBody":"{\"method\":\"eth_call\",\"params\":[{\"to\":\"0x221fa9cbafcd6c1c3d206571cf4427703e023ffa\",\"data\":\"0xe445e7dd\"},\"latest\"],\"id\":121,\"jsonrpc\":\"2.0\"}","requestMethod":"POST","url":"https://alfajores-forno.celo-testnet.org"}, code=CALL_EXCEPTION, version=providers/5.7.2)
```
</details>

### CODE REVIEW

**Comment by 0xmichalis** [comment]
Dispatching from Holesky to Base Sepolia seems to have worked:
```console
$ hyperlane send message --relay
Hyperlane CLI
? Select the origin chain holesky
? Select the destination chain basesepolia
Running pre-flight checks for chains...
✅ Chains are valid
✅ Signer is valid
All providers failed for method getGasPrice and params {}
✅ Balances are sufficient
Dispatching message
Pending https://holesky.etherscan.io/tx/0x29baa4f094f770eece75465f7fefa88467d5e799b17feb5183508beb018e2b63 (waiting 1 blocks for confirmation)
Sent message from holesky to 0xb7A35D65b478Ab08F3570838B6BbaD324692583e on basesepolia.
Message ID: 0x704d0387ad74520ac3ee27c14e7c64c98d0c237bcb4d8d7961b34ec4799bb16e
Message:
    parsed:
      version: 3
      nonce: 33
      origin: 17000
      sender: "0x000000000000000000000000a90ede59ec974adfa6f6a1f7acfd1d42089d8f23"
      destination: 84532
      recipient: "0x000000000000000000000000b7a35d65b478ab08f3570838b6bbad324692583e"
      body: "0x48656c6c6f21"
      originChain: holesky
      destinationChain: basesepolia
    id: "0x704d0387ad74520ac3ee27c14e7c64c98d0c237bcb4d8d7961b34ec4799bb16e"
    message: "0x030000002100004268000000000000000000000000a90ede59ec974adfa6f6a1f7a\
      cfd1d42089d8f2300014a34000000000000000000000000b7a35d65b478ab08f3570838b6bbad\
      324692583e48656c6c6f21"
    
Attempting self-relay of message
Pending https://sepolia.basescan.org/tx/0x677d5fbaead4956e05009075e0c1e8d51d401d53d580d1199fbeddfe4c211f9c (waiting 1 blocks for confirmation)
Message was self-relayed!
```

File: chains/basesepolia/addresses.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+staticMerkleRootMultisigIsmFactory: "0xEED8918B9Fb2AEcf2df79f1E07aA77EaC1017dAE"
```
Comment (jmrossy) [inline chains/basesepolia/addresses.yaml:1]: Please order these alphabetically to make the linter happy. I believe running `yarn lint:fix` would handle this for you.

File: chains/basesepolia/metadata.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - apiUrl: https://api-sepolia.basescan.org/api
+    family: etherscan
+    name: BaseScan
+    url: https://sepolia.basescan.org/
```
Comment (jmrossy) [inline chains/basesepolia/metadata.yaml:6]: Nit: avoid trailing slashes in these URLs please :)

**Review by jmrossy — COMMENTED** [review]
Thanks @0xmichalis for the contribution!!
You'll need a few more tweaks but this is almost ready. Please add a logo - it can be the same as base's logo - and see other comments.

File: chains/basesepolia/addresses.yaml (hunk 1 above)
Comment (0xmichalis) [reply chains/basesepolia/addresses.yaml]: updated

File: chains/basesepolia/metadata.yaml (hunk 2 above)
Comment (0xmichalis) [reply chains/basesepolia/metadata.yaml]: updated

**Review by jmrossy — APPROVED** [review]

**Comment by trachchung** [comment]
Thanks @0xmichalis , the new chain support is awesome, but it will be complete if you add more detail infomation like `InterchainGasPaymaster` `StorageGasOracle` and your mailbox default hook.

**Pull request merged**